### PR TITLE
[BUGFIX] Method emogrifyBodyContent() doesn't keeps utf8 umlauts

### PR DESCRIPTION
**Pull request by taiyonoryoshy**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 
 ### Fixed
+- Method emogrifyBodyContent() don't keeps utf8 umlauts
+  ([#347](https://github.com/jjriv/emogrifier/pull/347))
```
Comment (oliverklee) [inline CHANGELOG.md]: Could you please update this to 349 instead of 347?

File: CHANGELOG.md [hunk 2]
```diff
@@ -58,6 +58,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 
 ### Fixed
+- Method emogrifyBodyContent() don't keeps utf8 umlauts
```
Comment (oliverklee) [inline CHANGELOG.md]: don't -> doesn't

**Comment by taiyonoryoshy** [comment]
@oliverklee Yes, I have bad english :) Fix Your comments.

**Comment by oliverklee** [comment]
Merged. Thanks for the PR! ❤️

**Pull request merged**